### ISSUE TITLE
Unable to load Polyglot.CodeExecutor

### ISSUE THREAD

**Issue by pisani** [intersystems-community/vscode-iris-jupyter-server#5]
When attempting to define the Kernel - the message asking for permission to load "Polyglot.CodeExtractor" comes up, several times (about 5) but despite answering YES to load it, does not seem to download or install this.

Eventually, the top right-hand side of VS Code says 'ERROR @ irishealth:PYWORK'

my server name is irishealth, my namespace is PYWORK, and the Jupyter connection specified is http://localhost:50773/irishealth:PYWORK?token=1

**Comment by gjsjohnmurray** [comment]
Does your Activity Bar offer the InterSystems Tools button to get you to Server Manager's tree?

If so, does that tree include your `irishealth` server? And does its Namespaces subtree include the PYWORK namespace?

Does your `irishealth` server definition set the `username` property? And you used Server Manager's feature for storing that user's password in the workstation's keystore?

Please also report your server's $ZV string.

**Comment by gjsjohnmurray** [comment]
@pisani did you make any progress with this?

**Comment by pisani** [comment]
just tried it again @gjsjohnmurray ....  it works now.  I found when I loaded the class PolyglotKernel.CodeExecutor, it existed but did not compile since it had these lines:

  /// Derived from https://github.com/grongierisc/iris-python-template/blob/cdaacdf47c544b4f814a9ed4e2e1846ec0c0d441/src/ObjectScript/Kernel/CodeExecutor.cls
  Class PolyglotKernel.CodeExecutor
  {
  }

appended to the bottom fo the class, malforming it and making it un-compileable.   Not sure how this got added into the existing class.   We can close this issue unless you want to investigate how this happened.
